docs/UserGuide.adoc

```diff
@@ -819,43 +800,32 @@ Reps: 4 +
 Weight: 100 +
 Sets: 4 +
 |============
-You can use the `add-e` command to add the exercise as shown.
 
-. View the client that you want to add the exercise to. For information on how to view the client, you can refer to the `view-c` section. For this example, we will add the exercise to the first client in the list, `Alex Yeoh`. After you entered `view-c 1`, you will see `Alex Yeoh` detailed information and a table of his recorded exercises.
+. Use the `view-c` command to view "Alex Yeoh".
++
+
+. Type the information of the exercise, matching each detail of the exercise to the parameter. In this case, type `add-e n/Bench press d/12-02-2020 reps/4 ew/100 sets/4` into *Command Box*. Press enter to execute it.
 +
 image::ug-addeExamplePart1.png[]
-. Now type the information of the exercise, matching each detail of the exercise to the parameter. In this case, you will have to type `add-e n/Bench press d/12-02-2020 reps/4 ew/100 sets/4` into *Command Box*. Press enter to execute it. To recap,
+. After it has been successfully entered, *Result Box* will display that exercise has been sucessfully recorded. You will also be able to see the recorded exercise in the *Exercise Table*.
```

Comment:
> After it has been successfully entered

After the command has been successfully executed

> **Result Box** will display that exercise has been sucessfully recorded

**Result Box** will notify that the exercise has been recorded